statsmodels/stats/proportion.py

```diff
@@ -30,10 +30,12 @@ def proportion_confint(count, nobs, alpha=0.05, method='normal'):
         method to use for confidence interval,
         currently available methods :
 
-         - `normal` : asymptotic normal approximation
+         - `normal` : asymptotic normal approximation, or Wald method
+         - `normal_cc` : asymptotic normal approximation with continuity correction (cc)
          - `agresti_coull` : Agresti-Coull interval
-         - `beta` : Clopper-Pearson interval based on Beta distribution
+         - `beta` : Clopper-Pearson interval based on Beta distribution, or Exact method
          - `wilson` : Wilson Score interval
+         - `wilson_cc` : Wilson Score interval with continuity correction
```

Comment:
I think API with separate methods is fine. The alternative of adding a "continuity correction" keyword wouldn't apply generally enough.